cpp/src/parquet/column_writer_test.cc

```diff
@@ -373,7 +375,6 @@ class TestPrimitiveWriter : public PrimitiveTypedTest<TestType> {
 
   const ColumnDescriptor* descr_;
 
- private:
```

Comment:
Mark them `protected` if you just want to inherit them?

Reply:
The code here is:

```
 protected:
   ...
 private:
   ...
```

After remove private, they're already protected